packs/feats/invoke-the-elements.json

```diff
@@ -32,7 +32,59 @@
             "remaster": false,
             "title": "Pathfinder Lost Omens: Ancestry Guide"
         },
-        "rules": [],
+        "rules": [
+            {
+                "choices": [
+                    {
+                        "label": "PF2E.SpecificRule.Changeling.Brine.Label",
+                        "predicate": [
+                            "feat:brine-may"
+                        ],
+                        "value": "bludgeoning"
+                    },
+                    {
+                        "label": "PF2E.SpecificRule.Changeling.Snow.Label",
+                        "predicate": [
+                            "feat:snow-may"
+                        ],
+                        "value": "cold"
+                    },
+                    {
+                        "label": "PF2E.SpecificRule.Changeling.Veil.Label",
+                        "predicate": [
+                            "feat:veil-may"
+                        ],
+                        "value": "fire"
+                    },
+                    {
+                        "label": "PF2E.SpecificRule.Changeling.Virga.Label",
+                        "predicate": [
+                            "feat:virga-may"
+                        ],
+                        "value": "electricity"
+                    }
```

Comment:
```suggestion
                    {
                        "label": "PF2E.SpecificRule.Changeling.Lineage.Brine",
                        "predicate": [
                            "feat:brine-may"
                        ],
                        "value": "bludgeoning"
                    },
                    {
                        "label": "PF2E.SpecificRule.Changeling.Lineage.Snow",
                        "predicate": [
                            "feat:snow-may"
                        ],
                        "value": "cold"
                    },
                    {
                        "label": "PF2E.SpecificRule.Changeling.Lineage.Veil",
                        "predicate": [
                            "feat:veil-may"
                        ],
                        "value": "fire"
                    },
                    {
                        "label": "PF2E.SpecificRule.Changeling.Lineage.Virga",
                        "predicate": [
                            "feat:virga-may"
                        ],
                        "value": "electricity"
                    }
```
Though, I think we can maybe just skip these new labels and just use the damage types since users will never see them.